src/common/cert_vfy.c

```diff
@@ -467,7 +467,7 @@ int verify_certificate(X509 * x509, cert_policy *policy)
   X509_STORE_CTX *ctx = NULL;
 
   /* if neither ca nor crl check are requested skip */
-  if ( (policy->ca_policy==0) && (policy->crl_policy==CRLP_NONE) ) {
+  if ( (!policy->no_ca_policy==0) && (policy->crl_policy==CRLP_NONE) ) {
```

Comment:
I think it would be more readable to write this as
```suggestion
  if ( (policy->no_ca_policy==1) && (policy->crl_policy==CRLP_NONE) ) {
```